server/src/test/java/io/crate/analyze/expressions/IntervalAnalysisTest.java

```diff
@@ -143,14 +164,8 @@ public void test_odd() throws Exception {
     @Test
     public void test_odd_milliseconds() throws Exception {
         var symbol = e.asSymbol("INTERVAL '101' MILLISECOND");
-        assertThat(symbol).isLiteral(new Period().withMillis(101)
-                                         .withPeriodType(PeriodType.yearMonthDayTime()));
-    }
-
-    @Test
-    public void test_odd_milliseconds_to_Second() throws Exception {
-        var symbol = e.asSymbol("INTERVAL '1.99' SECOND TO MILLISECOND");
-        assertThat(symbol).isLiteral(new Period().withSeconds(1).withMillis(990)
+        // @todo this is strange, but Postgres says so too: # select INTERVAL '101' MILLISECOND; ->  00:01:41
+        assertThat(symbol).isLiteral(new Period().withMinutes(1).withSeconds(41)
```

Comment:
Because it's synonym of `select INTERVAL '101' as MILLISECOND` ie it's a column name.

To get 101 millis, we should put keyword inside quotes: `select INTERVAL '101 MILLISECOND' ;`

Reply:
Got it - I added another assert so that both cases are illustrated.